packages/tamagui/src/index.tsx

```diff
@@ -120,9 +129,18 @@ export const TamaguiComponents: ComponentRegistry = {
   SelectItemText: {
     component: t.Select.ItemText,
   },
+  SelectItemIndicator: {
+    component: t.Select.ItemIndicator,
+  },
   SelectSheet: {
     component: t.Select.Sheet,
   },
+  SelectContent: {
+    component: t.Select.Content,
+  },
+  SelectValue: {
+    component: t.Select.Value,
```

Comment:
chore: Some components were missing. Might be more, as Tamagui is quite large. We will be monitoring.

Reply:
yeah I wish we had an easy way to monitor Tamagui as more get added